src/integration/stimulus/testsuites/caliptra_top_nightly_directed_regression.yml

```
@@ -17,6 +17,7 @@ contents:
         - ${CALIPTRA_ROOT}/src/integration/test_suites/smoke_test_sha256_wntz_rand/smoke_test_sha256_wntz_rand.yml
         - ${CALIPTRA_ROOT}/src/integration/test_suites/smoke_test_sha3/smoke_test_sha3.yml
         - ${CALIPTRA_ROOT}/src/integration/test_suites/smoke_test_sha3_regs/smoke_test_sha3_regs.yml
+        - ${CALIPTRA_ROOT}/src/integration/test_suites/smoke_test_sha3_externalmu/smoke_test_sha3_externalmu.yml
```

Please also add 
```suggestion
        - ${CALIPTRA_ROOT}/src/integration/test_suites/smoke_test_sha3_externalmu/smoke_test_sha3_externalmu.yml
        - ${CALIPTRA_ROOT}/src/integration/test_suites/smoke_test_sha3_interrupt/smoke_test_sha3_interrupt.yml
```
As this should have been done in #1026 but was missed.

---

Thanks, I've added this!